api/src/main/resources/application.yml

```diff
@@ -94,13 +99,24 @@ csm:
       reader:
         username: "storage_reader_username"
         password: "storage_reader_password"
+      writer:
```

Comment:
The addition of the writer account could be made in a separate PR instead of having it seems like a dependency to rabbitmq